spec/controllers/works/multiple_actions_spec.rb

```diff
@@ -124,6 +124,19 @@
       end
     end
 
+    context "when work parameters are invalid" do
+      let(:work_params) { { work: { summary: "a" * (ArchiveConfig.SUMMARY_MAX + 1) } } }
+
+      before do
+        put :update_multiple, params: params
+      end
+
+      xit "redirects to the user multiple work path with an error" do
```

Comment:
Why is this test disabled?

Reply:
Also doesn't seem to have a reason so I've enabled it

Reply:
Yes, oops, should now be properly enabled, and I've fixed the name